baybe/constraints/continuous.py

```
@@ -115,25 +130,61 @@ def to_botorch(
         Args:
             parameters: The parameter objects of the continuous space.
             idx_offset: Offset to the provided parameter indices.
+            batch_size: The batch size used for the recommendation. Necessary
+                for interpoint constraints as indices need to be adjusted.
+                Ignored by all other constraints.
 
         Returns:
             The tuple required by botorch.
+
+        Raises:
+            RuntimeError: When the constraint is an interpoint constraint but
```

I guess similarly, you should also raise an error if it is the other way round

---

Addressed in ed752e8eed6ea4ae539eccfc13fdffdce4c6803f